L/LLVM/Clang@9.0.1/build_tarballs.jl

```diff
@@ -0,0 +1,5 @@
+name = "Clang"
+libllvm_version = v"9.0.1+2"
```

Comment:
It's a bit annoying that we have to declare that everywhere. Means more places to change it and more places that I can miss.

Reply:
Note that we are building _exactly_ that version; including the build number.  That's why this is important.

Reply:
This information however isn't propagated to the JLL package in the `compat` section though

Reply:
Shoot, you're right.  Grrr.

Reply:
Okay, fixed, but it won't give as fine-grained control as I want.  We won't be able to control patch versions with this, which means that you may get a newer `clang` or `opt` than the `libLLVM` that shipped with Julia, if we add some patches.  That's _probably_ fine, but not great.